Dockerfile

```
@@ -1,34 +1,22 @@
-FROM openjdk:8u212-jdk-stretch AS builder
+FROM openjdk:8u212-jre-slim-stretch
 
-RUN mkdir -p /opt/cassy
-WORKDIR /opt/cassy
+WORKDIR /cassy
 
-COPY gradle ./gradle
-COPY build.gradle ./
-COPY gradlew ./gradlew
-COPY src ./src
+# The Docker context is created in build/docker by `./gradlew docker`
+COPY ./cassy.tar .
+RUN tar xf cassy.tar -C / && rm -f cassy.tar
 
-RUN ./gradlew installDist
-
-# Runtime Container
-FROM openjdk:8-alpine
-
-RUN apk add sqlite
-
-WORKDIR /opt/cassy
-
-RUN mkdir -p /cassy/data && mkdir -p /cassy/conf
-
-# Copy Build output from Builder Step
-COPY --from=builder /opt/cassy/build ./build
 COPY scripts ./scripts
-COPY conf ./conf
+COPY logback.xml entrypoint.sh ./
 
-RUN sqlite3 ./cassy.db < ./scripts/db.schema
+RUN apt-get update && apt-get install -y \
+    sqlite \
```

If only use sqlite3. (☝️ Both version 2 and 3 will be included.)

```suggestion
    sqlite3 \
```

---

Thanks! Applied the change.